src/main/java/TestExecute.java

```diff
@@ -0,0 +1,122 @@
+import lombok.SneakyThrows;
+import org.telegram.telegrambots.meta.api.methods.polls.SendPoll;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.polls.Poll;
+import org.telegram.telegrambots.meta.api.objects.polls.PollAnswer;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestExecute extends Thread {
+
+    private final String botUsername;
+    private final String botToken;
+    private long chatId;
+
+    TestExecute(String botUsername, String botToken,long chatId) {
+        this.botUsername = botUsername;
+        this.botToken = botToken;
+        this.chatId = chatId;
+    }
+
+    @SneakyThrows
+    @Override
+    public void run() {
+        TestService test = new TestService(chatId);
+        EnglishForEveryDayBot engBot = new EnglishForEveryDayBot(botUsername,botToken);
+        PostgreSQLJDBS postgreSQLJDBS = new PostgreSQLJDBS();
```

Comment:
U create one + connection to database. why u don't close connection. Maybe u can use singleton